services/apps/alcs/src/providers/typeorm/migrations/1683048988258-generate_next_resolution_number.ts

```diff
@@ -8,33 +8,42 @@ export class generateNextResolutionNumber1683048988258
       `CREATE OR REPLACE FUNCTION alcs.generate_next_resolution_number(p_resolution_year integer)
           RETURNS integer
           LANGUAGE plpgsql
-        AS $function$
-          
-            declare next_resolution_number integer;
-          BEGIN	
+      AS $function$
```

Comment:
I do not think that this is going to be executed since the timestamp for migration did not change

Reply:
ah oops, looks like I created a new one and updated the old one

Reply:
Reverted